cargo-generate.toml

```
@@ -2,5 +2,6 @@
 include = ["Cargo.toml", "Cargo.lock", ".github/workflows/*", "*.rs"]
 
 [placeholders.itch_username]
-prompt = "Enter your itch.io username"
+prompt = "Enter your itch.io username. (Leave Blank to disable itch.io upload support)"
 type = "string"
+default = ""
```

```suggestion
default = ""

```
Do you not have `.editorconfig` support enabled in your IDE? If you're using VSCode, I advise you to download the extension ;)

---

I use RustRover

---

@Sapein in that case, could you activate the [EditorConfig](https://www.jetbrains.com/help/idea/editorconfig.html) plugin? That way, we all use the same formatting style :)